### DISCUSSION TITLE
Maintainers?

### DISCUSSION THREAD

**Discussion by Brian-McBride** [anatine/zod-plugins#150] [Announcements]
Looking to add more people to the repo to help maintain it. 
Any interests?

**Comment by Brian-McBride** [comment]
# Maintainers Needed

## Help?

My day job has kept me locked down hard. 

I'd love some help if anyone wants to maintain and help grow this repo.

## Objectives

This is a soft list of the overall objectives that would be great to accomplish

- Groom issues
- Look for areas to decrease bundle size
- Break common logic across Zod plugins into a core plugin module
- Migrate to the latest Nx (has better release management tooling)
- Implement conventional commits for PR requests, and overall clean up PR request process

**Reply by dandean** [reply]
@Brian-McBride hey there – I'd be interested in contributing to this in my spare time. Let me know!
